cpp/cmake_modules/BuildUtils.cmake

```diff
@@ -15,6 +15,20 @@
 # specific language governing permissions and limitations
 # under the License.
 
+# Common path suffixes to be searched by find_library or find_path.
+# Windows artifacts may be found under "<root>/Library", so
+# search there as well.
+set(LIB_PATH_SUFFIXES
+    "${CMAKE_LIBRARY_ARCHITECTURE}"
+    "lib64"
+    "lib32"
+    "lib"
+    "bin"
+    "Library",
```

Comment:
Remove the `,` and below.

Reply:
Oh shoot. Let me follow up with another PR to clean that up. Thanks for catching this.

Reply:
Never mind. I have a couple of changes to follow up. I can handle this.

Reply:
Awesome, thank you.